html/login.html

```diff
@@ -1,19 +1,19 @@
-<div>
-    <b>Bitte gib deine Nutzerdaten ein:</b>
+<br><br><br><br><br>
+<h1>Willkommen!</h1>
+<br>
+<h2>Bitte melde dich mit deinen Nutzerdaten an.</h2>
+<br><br><br><br>
+<div class="division">
+    <div class="box dark main_box" style="grid-row: 1; grid-column:3/11">
+        <form id="loginForm" method="post" action="/api" style="grid-row-gap:20px">
+            <input name="userName" class="text_input" id="userNameInput" placeholder="Nutzername" style="grid-row:2; grid-column:2/12;">
 
-    <form method="post" action="/api">
-        <label>
-            Benutzername:
-            <br>
-            <input name="userName" id="userNameInput">
-        </label><br>
-        <label>
-            Passwort:
-            <br>
-            <input name="password" id="passwordInput" type="password">
-        </label><br>
-        <button type="button" onclick="Login()">OK</button>
-        <br>
-        <label id="errorLabel" style="display: none;">Ungültige Zugangsdaten</label>
-    </form>
+            <input name="password" class="text_input" id="passwordInput" type="password" placeholder="Passwort" style="grid-row:3; grid-column:2/12;">
+
+            <a class="button_big" onclick="Login()" style="grid-row: 4; grid-column: 3/11;">Anmelden</a>
+
+            <label id="errorLabel" style="display: none; grid-row: 5;grid-column: 2/11;">Ungültige Zugangsdaten</label>
```

Comment:
This does not match the style. It should be centered and have more space below it.